src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -106,6 +106,9 @@ public bool Equals(AnalyzerReference? other)
         public override int GetHashCode()
             => Hash.Combine(RuntimeHelpers.GetHashCode(_assemblyLoader), FullPath.GetHashCode());
 
+        public override string ToString()
```

Comment:
Just to be sure; this was only to make your life easier when you were debugging something, right? There's no actual dependency on this format?

Reply:
Yes, this just makes debugging easier.

Reply:
Consider, rather than exposing a public override of `ToString`, using `DebuggerDisplayAttribute`. Not a big deal either way.

Reply:
I would prefer exposing `ToString` as it also improved some details of error messages that might appear in VS gold bars IIRC when I was debugging this a few months back.